### PR TITLE
Add Command Line Interface Pages project support

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** latest

## Notes

TlDr pages for other CLIP project tools will be added later. Accepting this PR will help my project gain more popularity, thanks for understanding.

image

### CODE REVIEW

File: pages/common/clip-view.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# clip-view
+
+> [C]ommand [L]ine [I]nterface [P]ages render.
+> Render for TlDr-like project with much a more extensive syntax and several render modes.
```
Comment (kbdharun) [inline pages/common/clip-view.md:4]:
```suggestion
> Render for a TlDr-like project with much more extensive syntax and several render modes.
```

**Review by kbdharun — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
@navarroaxel, can u review this PR?

File: pages/common/md-to-clip.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# md-to-clip
+
+> Converter from TlDr pages to [C]ommand [L]ine [I]nterface [P]ages.
+> More information: <https://github.com/command-line-interface-pages/prototypes/tree/main/md-to-clip>.
+
+- Convert specific local pages and output outcome pages in the same directories:
```
Comment (navarroaxel) [inline pages/common/md-to-clip.md:6]:
```suggestion
- Convert tldr-pages files and save into the same directories:
```

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
